internal/services/authorization/parse/role_assignment_test.go

```diff
@@ -166,6 +175,15 @@ func TestRoleAssignmentID(t *testing.T) {
 			},
 		},
 
+		{
+			// valid at subscriptions aliases scope
+			Input: "/providers/Microsoft.Subscription/aliases/my-awesome-sub/providers/Microsoft.Authorization/roleAssignments/23456781-2349-8764-5631-234567890121",
+			Expected: &RoleAssignmentId{
+				IsSubscriptionAliasLevel: true,
+				Name:                     "23456781-2349-8764-5631-234567890121",
```

Comment:
It looks like `IsSubscriptionAliasLevel` is not being tested below?

Also I think `SubscriptionAlias` should be added/tested here?

```suggestion
				Name:                     "23456781-2349-8764-5631-234567890121",
				SubscriptionAlias:        "my-awesome-sub",
```